### ISSUE TITLE
Add instructions for how to use it

### ISSUE THREAD

**Issue by Nytra** [MonkeyModdingTroop/ReferencePackageGenerator#1]
This is recommended for building the local nuget package feed, but there are no instructions given for how to use it. It would be helpful if these could be added. This might also include the nuget commands to setup the feed properly in VS.

**Comment by Banane9** [comment]
Yea, that'd probably be good xD

**Comment by iamgreaser** [comment]
Any progress on this? As is usually the case I'm not actually asking the question, I'm actually saying I could actually do with a quick guide on this.

Linux user here, specifically trying to build a Resonite mod which has no releases yet.

**Comment by Nytra** [comment]
I will try to get @Banane9 to write some instructions, but basically if you run the program once it should generate a config file which you can then edit with your paths and other options, it can then generate the packages from the targetted assemblies

**Comment by Banane9** [comment]
@Nytra's summary is correct. I'm currently at a convention, so I can't really write anything much right now.

Maybe @TumnusB's repo here could help too: https://github.com/TumnusB/ReferencePackageGenerator

**Comment by TumnusB** [comment]
Yeah, it's a little unprofessional but got someone going over it for me so soon it should be a lot more readable.
